packages/SketchMorph2-Tests.package/MorphicMonetTests.class/instance/testDoColorMorphsSetPenColor.st

```diff
@@ -0,0 +1,7 @@
+changing state tests
+testDoColorMorphsSetPenColor
+	| colorMorphs |
+	colorMorphs := self subject findByCriteria: [:curMorph |
+		curMorph isM2ColorMorph].
+	colorMorphs leftClickAssert: [:curMorph |
+		(self subject backend penColor diff: curMorph offColor ) < 0.01]
```

Comment:
This is out of the scope of this PR, but I think we should make an extension on `Color` testing for the diff being less than 0.01 because we use this so often.